src/oemof/solph/custom/sink_dsm.py

```diff
@@ -15,7 +15,9 @@
 SPDX-License-Identifier: MIT
```

Comment:
You might want to change the copyright to name our contribution.

Reply:
Right, thanks! It's pushed now. Thanks very much for approving and advising on this one!